mattermost/mattermost
# [MM-22872] fix(messaging): DM list not scrolling when you press the up/down arrow keys

#### Summary

This PR enables the DM list to scroll up or down when the user navigates to an option out of view.

#### Ticket Link


Fixes https://github.com/mattermost/mattermost/issues/26117
Jira https://mattermost.atlassian.net/browse/MM-22872

#### Screenshots


<details><summary>Before</summary>

https://github.com/user-attachments/assets/2453e889-8565-45dd-891d-1e0e0984e514

</details> 

<details><summary>After</summary>

https://github.com/user-attachments/assets/53a3c03e-a4fe-4e03-8a67-53186f5ef68a

</details> 

#### QA Steps

1. Start a DM with about 8 members (or any number large enough for you to scroll through).
2. Press the ↓ key until you reach the end of the list.
3. Verify the list scrolls down so each selected item is visible.
4. Once you reach the end, press the ↑ key until you reach the beginning.
5. Verify the list scrolls up so each selected item is visible.

#### Release Note

```release-note
Fixed scrolling behavior when navigating DM list using UP/DOWN arrow keys.
```
